tests/config/default-config-staging.test.js

```diff
@@ -28,11 +28,11 @@ describe('default-config', () => {
     process.env.AWS_DEFAULT_REGION = 'eu-west-1';
     process.env.AWS_REGION = 'eu-west-1';
     process.env.RDS_SANDBOX_ID = 'default';
-    process.env.AWS_ROLE_ARN = 'arn:aws:iam::242905224710:role/api-role-production';
+    process.env.AWS_ROLE_ARN = 'arn:aws:iam::123456789:role/api-role-production';
```

Comment:
I changed these hardcoded values for the tests as there doesn't seem to be any need for them to be organisation related

Reply:
if it's only test hcanges I think it sohuld be ok, but if there is any changes you want to perform that affect the platform itself it might be a good idea to raise the PR first in biomage-org and htem merge from there to this branch so that we don't end up with divergent repos.

The idea (as I understood it) of these PRs is that we are merging what is in our biomage-org `master` to hms's `master`, direct changes to this branch make it no longer part of biomage-org `master`)

Reply:
apologies for that -- I won't commit to these set of PRs directly